clang/test/CodeGenHLSL/builtins/select.hlsl

```diff
@@ -0,0 +1,41 @@
+// RUN: %clang_cc1 -finclude-default-header -x hlsl -triple \
+// RUN:   dxil-pc-shadermodel6.3-library %s -emit-llvm -disable-llvm-passes \
+// RUN:   -o - | FileCheck %s --check-prefixes=CHECK
+
+// CHECK: %hlsl.select = select i1
+// CHECK: ret i32 %hlsl.select
```

Comment:
It's useful to put a `CHECK:` or a [`CHECK-LABEL:`](https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-label-directive) to anchor us into the right function. This makes it less likely for one of the checks to go find something from another function, resulting in a very confusing error message. Here we should be able to use `CHECK-LABEL: test_select_bool_int` and the equivalent for each function.